W-15596023-CIDR-changes (#199)
mulesoft/docs-hosting: cloudhub-2/modules/ROOT/pages/ps-gather-setup-info.adoc

MuleSoft recommends using subnets from standard RFC 1918 addresses, such as `10.0.0.0/8`, `172.16.0.0/12`, and `192.168.0.0/16`.

[[reserved-cidr]]

> ==== Reserved and Unusable CIDR

Reviewer: Unusable CIDR.

We don't want to mix reserved and unusable CIDRs.

Author: ```suggestion
==== Unusable CIDR
```

> You cannot use the following list of CIDR blocks for reserved CIDR blocks or as private space network CIDRs, as these are reserved by Anypoint Platform:

Reviewer: You cannot use the following list of CIDR blocks as private space network CIDRs:

Author: ```suggestion
You cannot use the following list of CIDR blocks as private space network CIDRs, as these are reserved by Anypoint Platform:
```

Reviewer: I don't think we need to make any change here regarding reserved CIDR feature, let's restore to its original content..

Author: Delete it or go with my original text?

Reviewer: can we restore it to original text before the change for reserved cidrs?

Author: ```suggestion
You cannot use the following list of CIDR blocks:
```

MuleSoft recommends using subnets from standard RFC 1918 addresses, such as `10.0.0.0/8`, `172.16.0.0/12`, and `192.168.0.0/16`.

[[reserved-cidr]]

> ==== Reserved CIDR

Reviewer: Can we change to "Optional Reserved Corporate CIDRs"?

Author: ```suggestion
==== Reserved Corporate CIDR
```

Author: I don't think optional should be a lead-in here, but I'll remind them in the text.

> When you create your private space and configure your private network, you can configure _reserved CIDR blocks_ that correspond with your reserved corporate network CIDRs. When you reserve CIDRs in private network configuration, Anypoint Platform does not use the blocks you specify when associating IP addresses with your private space. This prevents overlap between your corporate network CIDRs and Anypoint Platform infrastructure. 

Author: ```suggestion
Optionally, when you create your private space and configure your private network, you can configure _reserved CIDR blocks_ that correspond with your reserved corporate network CIDRs. When you reserve CIDRs in private network configuration, Anypoint Platform does not use the blocks you specify when associating IP addresses with your private space. This prevents overlap between your corporate network CIDRs and Anypoint Platform infrastructure. 
```

> Optionally, when you create your private space and configure your private network, you can configure _reserved CIDR blocks_ that correspond with your reserved corporate network CIDRs. When you reserve CIDRs in private network configuration, Anypoint Platform does not use the blocks you specify when associating IP addresses with your private space. This prevents overlap between your corporate network CIDRs and Anypoint Platform infrastructure. 

Reviewer 2: ```suggestion
Optionally, when you create your private space and configure your private network, you can configure _reserved CIDR blocks_ that correspond with your reserved corporate network CIDRs. When you reserve CIDRs in a private network configuration, Anypoint Platform does not use the blocks you specify when associating IP addresses with your private space. This prevents overlap between your corporate network CIDRs and Anypoint Platform infrastructure. 
```

> There are two strategies for reserving CIDRs during private space creation:

Reviewer: these look good!

Author: THANK YOU! for your patience with me :)))

169.254.169.252/30
----

> You can configure the following private space internal CIDRs as reserved CIDR blocks, but you cannot use them as private space network CIDRs:

Reviewer: let's document the list as "MuleSoft infrastructure CIDRs pool".

Author: ```suggestion
You can configure the following CIDRS from the MuleSoft infrastructure CIDRs pool as reserved CIDR blocks, but you cannot use them as private space network CIDRs:
```

> You can configure the following CIDRS from the MuleSoft infrastructure CIDRs pool as reserved CIDR blocks, but you cannot use them as private space network CIDRs:

Reviewer: Anypoint platform uses the following CIDRs as internal CIDR pool, they maybe updated later if needed.

Author: ```suggestion
Anypoint Platform has a list of CIDRs as an internal CIDR pool for provisioning private space networks that may be subject to change. You can configure the following as reserved CIDR blocks, but you cannot use them as private space network CIDRs:
```

Reviewer: Anypoint Platform has a list of CIDRs as an internal CIDR pool for provisioning private space network, they may be subject to change. You can configure them as reserved corporate CIDR blocks, but you cannot use them as private space network CIDRs:

----
100.64.0.0/16
100.66.0.0/16
100.67.0.0/16
100.68.0.0/16
----

> Additionally, AWS EKS uses `10.100.0.0/16` or `172.20.0.0/16` inside of private space networks. 

Reviewer: let's come back to this later after reserved CIDR is documented.

Author: ```suggestion
// Additionally, AWS EKS uses `10.100.0.0/16` or `172.20.0.0/16` inside of private space networks. 
```

169.254.169.252/30
----

> Anypoint Platform has a list of CIDRs as an internal CIDR pool for provisioning private space networks. The MuleSoft CIDR pool is subject to change. You can configure the following as reserved CIDR blocks, but you cannot use them as private space network CIDRs:

Reviewer: can we change to "You can configure some of the following as reserved CIDR blocks"?

Author: ```suggestion
Anypoint Platform has a list of CIDRs as an internal CIDR pool for provisioning private space networks. The MuleSoft CIDR pool is subject to change. You can configure some of the following as reserved CIDR blocks, but you cannot use them as private space network CIDRs:
```

----
100.64.0.0/16
100.66.0.0/16
100.67.0.0/16
100.68.0.0/16

> ----

Author: @tonylsw the four entries are down here.

Reviewer: yes, they look good here.
